burger-app/src/components/Main/Burger/index.js

```diff
@@ -1,13 +1,37 @@
 import styled from "styled-components";
 import Image from "../image";
 
-const Burger = ({ orderPrice }) => {
+const Burger = ({ orderPrice, orderIngredients, ingredientsInOrder, checkout }) => {
   return (
     <BurgerWrapperStyled>
       <TotalPriceStyled>Total: {orderPrice}₴</TotalPriceStyled>
       <Image ingredient="top_bun" alt="Top Bun" />
-      <SpanStyled>Please, start by adding products...</SpanStyled>
+      {orderIngredients ? (
+        <SpanStyled />
+      ) : (
+        <SpanStyled>Please, start by adding products...</SpanStyled>
+      )}
```

Comment:
I think it also should work like this, looks a little shorter 
```suggestion
      {orderIngredients && <SpanStyled>Please, start by adding products...</SpanStyled>}
```